src/app/components/Text/index.stories.tsx

```diff
@@ -1,91 +1,79 @@
 import React from 'react';
-import { withKnobs, select } from '@storybook/addon-knobs';
 
-import { ServiceContextProvider } from '../../contexts/ServiceContext';
-import { withServicesKnob } from '../../legacy/psammead/psammead-storybook-helpers/src';
-import ThemeProvider from '../ThemeProvider';
+import { FontVariant, GelFontSize } from '#app/models/types/theming';
 import Text from '.';
-import md from './README.md';
+import readme from './README.md';
 import { StoryProps } from '../../models/types/storybook';
 
 interface Props extends StoryProps {
   text: string;
+  fontVariant: FontVariant | '--';
+  size: GelFontSize | '--';
 }
-
 const EMPTY_OPTION = '--';
```

Comment:
As above, can you use `EMPTY_OPTION` in the type definitions of fontVariant & size?